v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -716,8 +721,6 @@ func (r *grpcReporter) eventBatchSender(batches <-chan [][]byte) {
 	for {
 		// this will block until a message arrives or the reporter is closed
 		select {
-		case b := <-batches:
-			messages = b
```

Comment:
what was this `case b` used for?

Reply:
It does the same thing as the next case (to received batched messages from the channel `batches`), except that it doesn't check the length, therefore it doesn't quit when the channel is closed. For some reasons, it didn't get removed in the previous PRs.